### DISCUSSION TITLE
openRV Icon

### DISCUSSION THREAD

**Discussion by rubikstriangle** [AcademySoftwareFoundation/OpenRV#473] [Ideas]
How about an icon that honors the original tweak RV icon, this time the RV is open.

![openRV_Logo](https://github.com/AcademySoftwareFoundation/OpenRV/assets/42789620/ffca4623-3073-4f81-a80d-fd8a46b8c60d)

**Comment by bernie-laberge** [comment]
Wow ! It is beautiful !!!

**Comment by rogernelson** [comment]
Amazing work!

**Comment by wgergely** [comment]
Hello everyone,

I'll just chime in as someone who uses RV (and OpenRV recently!).

I love the app and rely on it a great deal to help me review footage. Of the many strengths of the app the UI is not one of them and I'd love to see some work on it to bring it into line with recent DCC ui trends - this would include the logo I suppose!

Whilst that proposed image is interesting, I think there's a strong argument to be made in favour of something more minimal, simpler, visually easier to parse. On Windows, when RV is made the default image reader the app's icon is the default icon for all associated image formats and the current OpenRV icon is a bit too convoluted.

I wonder if something simpler, even text based (somewhere along the lines of how Adobe handles app icons) would help convey the professionalism of the app a bit more? Just my two cents ✌️

**Comment by o-Dubbie-o** [comment]
This is AI slop. no thanks.
